src/events/rawChannelDelete.ts

```diff
@@ -0,0 +1,78 @@
+import { Snowflake } from '@klasa/snowflake';
+import { DbSet } from '@lib/structures/DbSet';
+import { WSChannelDelete } from '@lib/types/DiscordAPI';
+import { Events } from '@lib/types/Enums';
+import { GuildSettings } from '@lib/types/settings/GuildSettings';
+import { APIErrors, Time } from '@utils/constants';
+import { api } from '@utils/Models/Api';
+import { resolveOnErrorCodes } from '@utils/util';
+import { Event, EventStore } from 'klasa';
+
+export default class extends Event {
+
+	public constructor(store: EventStore, file: string[], directory: string) {
+		super(store, file, directory, { name: 'CHANNEL_DELETE', emitter: store.client.ws });
+	}
```

Comment:
Is it not the standard t @applyOptions for pieces in this case?

Reply:
Can't due to the emitter